src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.Button;
+
+/**
+ * A wrapper class for our custom-made operator board. Does not have a
+ * constructor. Access statically. Access Buttons via public static state.
+ * Access axis values via methods. Use methods to make buttons light up.
+ */
+public class OperatorBoard {
+
+    private static Joystick HARDWARE;
+    public OperatorBoardButton BUTTON_ONE;
+    public OperatorBoardButton BUTTON_TWO;
+    public OperatorBoardButton BUTTON_THREE;
+    public OperatorBoardButton BUTTON_SIX;
+    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton BUTTON_TWELVE;
+    public OperatorBoardButton BUTTON_THIRTEEN;
+    public OperatorBoardButton BUTTON_FOURTEEN;
+
+    private static OperatorBoard instance;
+
+    /**
+     * The constructor is kept private so that I can make sure it is only run once,
+     * and that there is only ever one instance of OperatorBoard in existence.
+     */
```

Comment:
why are there 2 instances of this javadoc?

Reply:
Why not.  what I wrote is a good explanation for both the constructor method and the getInstance method

Reply:
the javadoc for getInstance should include @return and should say what it does